Source/AIExtensions/Private/DataProviders/AIExrAIDataProvider_GameplayAttribute.cpp

```diff
@@ -0,0 +1,29 @@
+#include "DataProviders/AIExrAIDataProvider_GameplayAttribute.h"
+
+#include "AbilitySystemBlueprintLibrary.h"
+#include "AbilitySystemComponent.h"
+
+void UAIExrAIDataProvider_GameplayAttribute::BindData( const UObject & owner, int32 request_id )
+{
+    if ( const auto * asc = UAbilitySystemBlueprintLibrary::GetAbilitySystemComponent( const_cast < AActor * >( Cast< AActor >( &owner ) ) ) )
```

Comment:
Why the double cast? could this be done with one?

Reply:
no. Cast does not allow to change constness